robottelo/utils/vault.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, env_file='.env'):
     def setup(self):
         if self.env_path.exists():
             self.envdata = self.env_path.read_text()
-            is_enabled = re.findall('\nVAULT_ENABLED_FOR_DYNACONF=(.*)', self.envdata)
+            is_enabled = re.findall('^(?:.*\n)*VAULT_ENABLED_FOR_DYNACONF=(.*)', self.envdata)
```

Comment:
Please replace it by 
`re.findall('^VAULT_URL_FOR_DYNACONF=(.*)', self.envdata, re.MULTILINE)` 

`^` and MULTILINE is what we need

Reply:
With the regex, it has now, it does not need `re.MULTILINE` also the `^` start of the line is covered already.

In future PR we will integrate DotEnv python lib as suggested by @ogajduse !